src/pseudopeople/noise_functions.py

```diff
@@ -183,7 +183,7 @@ def miswrite_numerics(
     noised_column = pd.Series("", index=column.index)
     digits = []
     for i in range(len(is_number.columns)):
-        digit = np.where(replace.loc[:, i], random_digits[:, i], same_len_col.str[i])
+        digit = np.where(replace.iloc[:, i], random_digits[:, i], same_len_col.str[i])
```

Comment:
Does this not work with `loc`? What are the column names?

Reply:
No because loc is name based.